Cargo.toml

```diff
@@ -100,3 +104,69 @@ syslog      = "7.0"
 [features]
 default = []
 tls = ["tokio-rustls", "rustls-pemfile"]
+
+# TODO: Do we want this?
+# [profile.release]
+# panic = "abort"
+
+[package.metadata.deb]
+assets = [
+	[
+		"target/release/cascade",
+		"usr/bin/",
+		"755",
+	],
+	[
+		"target/release/cascadec",
```

Comment:
While not caused by this PR, we should decide what we want the client binary to be called. `+c` is an approach we used for Krill, but NSD uses `nsd-control` and OpenDNSSEC uses `ods-control`, `ods-signer` and `ods-enforcer` with the daemons having `+d`, which we may want to do as well?

Reply:
Maybe we use cascaded for the daemon, then we can use cascade for the cli. cascade-control seems a bit long.

Reply:
`+d` is quite a normal thing in the Linux world though, I suppose a direct comparison like `knotc` and `knotd` fails because we don't say `cascaded` and `cascadec`, but the single character difference between c and d is both easy to mistype and prevents auto-complete so I prefer `cascade` without the `c` for that reason. This is also a reason against `xxx-control` because someone can accidentally invoke the daemon via `xxx` when they meant the client, so the client having a shorter name than the daemon reduces the chance of that foot gun.